frontend/js/src/settings/missing-data/MissingMBData.tsx

```diff
@@ -266,137 +287,120 @@ export default class MissingMBDataPage extends React.Component<
-                  currPage && currPage >= totalPages ? "hidden" : ""
-                }`}
-                style={{ marginLeft: "auto" }}
-              >
-                <a
-                  role="button"
-                  onClick={this.handleClickNext}
-                  onKeyDown={(e) => {
-                    if (e.key === "Enter") this.handleClickNext();
-                  }}
-                  tabIndex={0}
+                  <a
+                    role="button"
+                    onClick={this.handleClickPrevious}
+                    onKeyDown={(e) => {
+                      if (e.key === "Enter") this.handleClickPrevious();
+                    }}
+                    tabIndex={0}
+                  >
+                    &larr; Previous
+                  </a>
+                </li>
+                <li
+                  className={`next ${
+                    currPage && currPage >= totalPages ? "hidden" : ""
+                  }`}
+                  style={{ marginLeft: "auto" }}
                 >
-                  Next &rarr;
-                </a>
-              </li>
-            </ul>
+                  <a
+                    role="button"
+                    onClick={this.handleClickNext}
+                    onKeyDown={(e) => {
+                      if (e.key === "Enter") this.handleClickNext();
+                    }}
+                    tabIndex={0}
+                  >
+                    Next &rarr;
+                  </a>
+                </li>
+              </ul>
+            </div>
           </div>
+          <BrainzPlayer
+            listens={missingMBDataAsListen}
+            listenBrainzAPIBaseURI={APIService.APIBaseURI}
+            refreshSpotifyToken={APIService.refreshSpotifyToken}
+            refreshYoutubeToken={APIService.refreshYoutubeToken}
+            refreshSoundcloudToken={APIService.refreshSoundcloudToken}
+          />
         </div>
-        <BrainzPlayer
-          listens={missingMBDataAsListen}
-          listenBrainzAPIBaseURI={APIService.APIBaseURI}
-          refreshSpotifyToken={APIService.refreshSpotifyToken}
-          refreshYoutubeToken={APIService.refreshYoutubeToken}
-          refreshSoundcloudToken={APIService.refreshSoundcloudToken}
-        />
-      </div>
+      </>
     );
   }
 }
 
-document.addEventListener("DOMContentLoaded", () => {
-  const {
-    domContainer,
-    reactProps,
-    globalAppContext,
-    sentryProps,
-  } = getPageProps();
-  const { sentry_dsn, sentry_traces_sample_rate } = sentryProps;
-
-  if (sentry_dsn) {
-    Sentry.init({
-      dsn: sentry_dsn,
-      integrations: [new Integrations.BrowserTracing()],
-      tracesSampleRate: sentry_traces_sample_rate,
-    });
-  }
-
-  const { missingData, user } = reactProps;
+export function MissingMBDataPageWrapper() {
+  const data = useLoaderData() as MissingMBDataProps;
+  const { currentUser: user } = React.useContext(GlobalAppContext);
+  return <MissingMBDataPage {...data} user={user} />;
```

Comment:
I think this might need to be: 
```suggestion
  return <MissingMBDataPage missingData={data} user={user} />;
```

Reply:
Fixed